kpi/utils/data_exports.py

```diff
@@ -98,12 +176,48 @@ def flatten_settings_inplace(settings: dict) -> None:
             settings[k] = ''
 
 
-def get_row_value(row: dict, col: str) -> Union[str, int, float, bool, None]:
-    val = row.get(col, '')
-    # remove any new lines from text
-    if isinstance(val, str):
-        val = val.replace('\n', '')
-    return val
+def get_data(filtered_queryset: QuerySet, export_type: str) -> QuerySet:
```

Comment:
now that we have more cases, we should just have each type of ExportTask have its own method for getting data so we don't just keep adding `elif`s for every new kind of export task.